### PR TITLE
fixes 1179 - header not clickable

### PR DESCRIPTION
**Pull request by jberube**
Had to use a different shadow than the one in Zeplin to avoid having a shadow on top, without changin the z-index.

### CODE REVIEW

File: src/components/EntityPage/EntityActionBar.js [hunk 1]
```diff
@@ -19,10 +19,9 @@ const ActionBar = styled(Flex)`
   height: 62px;
   width: 100%;
 
-  box-shadow: 0 0 4.9px 0.1px ${({ theme }) => theme.lighterShadow};
+  box-shadow: 0 4.9px 4.9px -4.9px ${({ theme }) => theme.lighterShadow};
   border: solid 1px ${({ theme }) => theme.greyScale5};
   background-color: ${({ theme }) => theme.white};
-  z-index: -1;
```
Comment (nyanofthemoon) [inline src/components/EntityPage/EntityActionBar.js]: 🐙

Comment (ciaranschutte) [reply src/components/EntityPage/EntityActionBar.js]: What does octopus thingy mean?

**Comment by ciaranschutte** [comment]
lgtm

**Comment by hlminh2000** [comment]
vr preview link :) : https://5c5c91b1a374a900080a0a01--kf-vr-portal.netlify.com/

**Pull request merged**